.nocover.yaml

```diff
@@ -8,6 +8,7 @@ nocover_file_globs:
   - community/git.py
   - gci/*.py
   - gsoc/*.py
+  - gitter/nlp/dict_tagger.py
```

Comment:
use `def tag_sentence` in the regex section, if that is the problem.

but you must get full coverage of `__init__`